feat(content): dreammall - new text in German including a picture (#243)
IT4Change/IT4C.dev: docs/projects/dreammall.md

# DreamMall.earth

> ![dreammall.earth](../images/projects/dreammall.png)

Reviewer: The logo image is quite huge related to the page's heading etc.
What about using the Vuepress [Markdown-Image Plugin](https://ecosystem.vuejs.press/plugins/markdown/markdown-image.html) to resize images in our website the [vuepress way](https://ecosystem.vuejs.press/plugins/markdown/markdown-image.html#image-size)?

Incidentally, the plugin's parameter [mark](https://ecosystem.vuejs.press/plugins/markdown/markdown-image.html#image-mark) can be used to control the use of different images for the light/dark mode.

Author: Feel free to use that plugin and refactor the page - this is out of scope

> 

Reviewer: In order to keep the project descriptions coherent, we have given the description section the heading used so far.

```suggestion

## Beschreibung

```

Author: it bares no meaning

*DreamMall.earth* ist eine Video & Projekt Management Plattform, die als Open-Source-Software für Claudia Siebert und Sabine Laudien entwickelt wird. Das Projekt schafft einen *beweglichen Raum* für Unternehmer und Selbstständige ihre Ideen in die Realität zu tragen und Unterstützung von Gleichgesinnten zu finden.

> Das zentrale Element des Projektes ist eine *Sternenkarte*, welche die einzelnen Akteure und Projekte an einen fiktiven Sternenhimmel projiziert. Der Nutzer kann in Austausch mit Anderen gehen, in dem er seinen Raum öffnet und andere dazu einläd. Ein Raum ist eine Videokonferenz mit Chatfunktion & Bildschirmfreigabe.

Reviewer: ```suggestion
Das zentrale Element des Projektes ist eine *Sternenkarte*, welche die einzelnen Akteure und Projekte an einen fiktiven Sternenhimmel projiziert. Jeder Nutzer kann in Austausch mit Anderen gehen, in dem er seinen Raum öffnet und andere dazu einläd. Ein Raum ist eine Videokonferenz mit Chatfunktion, Bildschirmfreigabe, gemeinsamen Notizen, Whiteboard, Unterräumen und Vielem mehr.
```
Und wollen wir bereits an dieser Stelle darauf verweisen, dass die  Features in Räumen BigBlueButton Features sind?

Author: Nein, es gibt eine Tech section, die die verwendete Technologie beschreibt.

> In der Zukunft sind Projektmanagementwerkzeuge angedacht, die es dem Nutzer vereinfachen seine Projekte zu organisieren und andere daran teilhaben zu lassen.

Reviewer: ```suggestion
Das Projekt ist noch in Entwicklung.
Es sind für die Zukunft weitere Features zur Unterstützung der DreamMall Akteure bei Kommunikation und Projektumsetzung angedacht,  z.B. 
- Projektmanagementwerkzeuge
- Neuigkeiten zu Projekten
- Veranstaltungsverwaltung
- Suche & Angebot von Projekten und - mitwirkung
```

Author: Applied manually in an adjusted form

Mehr Infos auf [DreamMall.earth](https://dreammall.earth) und [GitHub](https://github.com/dreammall-earth/dreammall.earth).

## Techstack

> *DreamMall.earth* basiert auf der *Node.js*-Technologie. Es kommt *vue*, *vike*, *vuetify*, *apollo*, *graphql* und *prisma* zum Einsatz. Die Video Konferenzen werden mit *BigBlueButton* realisiert.

Reviewer: ```suggestion
*DreamMall.earth* basiert auf der *Node.js*-Technologie. Es kommen *Vue*, *Vike*, *Vuetify*, *Apollo*, *GraphQL* und *Prisma* zum Einsatz. Die Video Konferenzen werden mit *BigBlueButton* realisiert.
```
